lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -234,18 +232,18 @@ void StagesSubject::stageChanged(UsdNotice::ObjectsChanged const& notice, UsdSta
 					Ufe::Scene::notifyObjectDelete(notification);
 				}
 			}
+#ifdef UFE_V2_FEATURES_AVAILABLE
 			else
 			{
-#if UFE_PREVIEW_VERSION_NUM >= 2014
 				// According to USD docs for GetResyncedPaths():
 				// - Resyncs imply entire subtree invalidation of all descendant prims and properties.
 				// So we send the UFE subtree invalidate notif.
 				auto notification = Ufe::SubtreeInvalidate(sceneItem);
 				Ufe::Scene::notifySubtreeInvalidate(notification);
-#endif
 			}
+#endif
 		}
-#if UFE_PREVIEW_VERSION_NUM >= 2015
+#ifdef UFE_V2_FEATURES_AVAILABLE
```

Comment:
This UFE v2 block encompasses the ObjectDestroyed and SubtreeInvalidate which only exist in v2.